egs/swbd/s5/local/score.sh

```diff
@@ -31,7 +31,7 @@ data=$1
 
 if [ -f $data/stm ]; then # use sclite scoring.
   echo "$data/stm exists: using local/score_sclite.sh"
-  eval local/score_sclite.sh $orig_args
+  eval local/score_sclite.sh "$orig_args"
```

Comment:
I just noticed that all of these changes are in swbd/s5.  This is super outdated.  You should be using s5c.  I doubt that this problem (if there was a problem) occurs in the latest script.  In any case let me know what the problem was, because I'd be surprised if this was really a bug, this script being so old.

Reply:
IIRC if I don't add the "", if the $orig_args has something like --cmd "queue.pl --mem 8G" it'll complain.